lnst/Recipes/ENRT/BaseEnrtRecipe.py

```diff
@@ -404,6 +414,8 @@ def generate_perf_configurations(self, config):
             )
 
             self.register_perf_evaluators(perf_conf)
```

Comment:
Forgot to remove this line. WIll fix with another force push. This bug got confirmed in the job.